fetch/fetch.go

```diff
@@ -227,32 +260,34 @@ func (f *Fetch) loop() {
 			f.activeRequests[req.hash] = append(f.activeRequests[req.hash], req)
 			rLen := len(f.activeRequests)
 			f.activeReqM.Unlock()
+			f.log.Info("request added to queue %v", req.hash.ShortString())
 			if req.priority > Low {
 				f.sendBatch([]requestMessage{{req.hint, req.hash}})
```

Comment:
I understand we send high priority requests right away.. we're purposely not batching them?

Reply:
this was the initial thought yes
I can change that to batch some requests